Scripts/developer_scripts/create_new_release

```diff
@@ -180,9 +180,13 @@ else
 fi
 # Set the major/minor/bugfix release numbers
 NUMBERS_DIR=${SOURCES_DIR}/Maintenance/release_building
-MAJOR_NUMBER=`cat ${NUMBERS_DIR}/MAJOR_NUMBER`     # 2 digits max
-MINOR_NUMBER=`cat ${NUMBERS_DIR}/MINOR_NUMBER`     # 2 digits max
-BUGFIX_NUMBER=`cat ${NUMBERS_DIR}/BUGFIX_NUMBER`   # 1 digit max
+if [ -f ${NUMBERS_DIR}/MAJOR_NUMBER ]; then
+    MAJOR_NUMBER=`cat ${NUMBERS_DIR}/MAJOR_NUMBER`     # 2 digits max
+    MINOR_NUMBER=`cat ${NUMBERS_DIR}/MINOR_NUMBER`     # 2 digits max
+    BUGFIX_NUMBER=`cat ${NUMBERS_DIR}/BUGFIX_NUMBER`   # 1 digit max
+else
+    eval $(cmake -DCGALCONFIGVERSIONFILE=${SOURCES_DIR}/CGALConfigVersion.cmake -P ${SOURCES_DIR}/Scripts/developer_scripts/create_new_release_evaluate_versions.cmake 2>&1)
```

Comment:
Why is an include not sufficient?

Reply:
`create_new_release` is a shell script and `CGALConfigVersion.cmake` is a CMake script. They do no share any part of common syntax.

Reply:
Sorry, I thought it was the new cmake script to create a release. BTW, I thought that the cmake script was the one currently in use.

Reply:
We actually have three scripts:
- the CMake script `Scripts/developer_scripts/cgal_create_release_with_cmake.cmake` creates an internal release, or a public release, from a Git checkout,
- the shell script `Scripts/developer_scripts/create_new_release` creates both the internal release, the public release, and the Windows installer, and it calls the previous one,
- the shell script `Maintenance/infrastructure/cgal.geometryfactory.com/bin/create_release` is for me only: it hard-codes local Git repositories, do the `git pull`/`git push`, calls the previous script, and then public the tarballs and installer on the web site.